CVLByExamples/ReadOnlyReentrancy/certora/conf/BuiltinROReentrancyVulnerableBankFixed.conf

```diff
@@ -0,0 +1,16 @@
+{
+    "files": [
```

Comment:
can we make sure it works with certora-cli 4.3.1? or is it not yet?

Reply:
Will check after the release